### ISSUE TITLE
Inconsistencies in the design of built-in apps

### ISSUE THREAD

**Issue by erikinkinen** [capyloon/nutria#19]
Almost every app has a different-coloured background which fails to give the impression of consistent design.

**Environment**
- Desktop Linux
- Desktop Mac
- Device

**To Reproduce**
1. Open a lot of built-in apps.
2. Look at them
3. Notice the inconsistencies I described.

**Expected behavior**
Every app should have a similar design with for example a white background.

**Additional context**
The backgrounds are probably the most prominent inconsistency but there might be other ones as well hidden behind the prominence of the backgrounds.

**Comment by fabricedesre** [comment]
Right... I've been looking at re-using an existing set of UI components to have a better consistency (and quality!). I think [Shoelace](https://shoelace.style) could be a good option since https://github.com/shoelace-style/shoelace/issues/571 is closed.

I have a WIP branch for the First Time Use app where I'm testing it. Hopefully will be ready in the coming weeks, and based on how this goes we could migrate other apps too.

**Comment by erikinkinen** [comment]
@fabricedesre 
I started migrating settings to shoelace.
#22  is a draft PR where you can take a look at the work I've done already. I also attach the mock-ups I sent in the chat here.

![settings-mainmenu](https://user-images.githubusercontent.com/8597575/157238582-bd673365-f5d1-4704-ae8b-c29cf0219a76.jpg)

![settings-wifi](https://user-images.githubusercontent.com/8597575/157238614-2288a6ca-8853-48ae-9689-51770b7ca32d.jpg)
